GabrielaRamirez/GradleExample/workspace/src/main/java/hackkerank/day3/ransomNote/MagazineNoteCheck.java

```diff
@@ -0,0 +1,40 @@
+package hackkerank.day3.ransomNote;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class MagazineNoteCheck {
+
+    public static void checkMagazine(List<String> magazine, List<String> note) {
+
+        Map<String, Integer> magazineWords = new HashMap<>();
```

Comment:
It is recommended that the name of the collections and maps estart with singular name and them the name of the interface, example: numberList